source/test/TestApp/PlayFabEventTest.cpp

```diff
@@ -218,6 +218,16 @@ namespace PlayFabUnit
         std::bind(&PlayFabEventTest::NonStaticEmitEventCallback, this, std::placeholders::_1, std::placeholders::_2));
     }
 
+    void PlayFabEventTest::MutliThreadTest(TestContext& testContext)
+    {
+        //TODO: spin up X threads (2 or 10)
+        // Each thread should send 10 events
```

Comment:
considering the nature of this change, this TODO seems pretty critical to resolve before checking anything in.